config/locales/en.yml

```diff
@@ -2924,6 +2924,7 @@ en:
           title: Here are the income forms we transferred from your federal tax return.
           unemployment_title: Unemployment benefits (1099-G)
           w2s_title: Jobs (W-2)
+          warning: "⚠️ We need to double-check some information"
```

Comment:
[dust] are there any potential issues with using an emoji here instead of an svg or something?

Reply:
I don't think it's an issue for this project, but it's a little bit up to the browser how exactly to render the emoji as opposed to an image we know will always look exactly the same. And for accessibility we don't want to rely on it, but we are ok since the rest of our text gives the actual information. So using this should be all good.